README.md

```diff
@@ -45,19 +45,29 @@ To run locally `npm run storybook`, it will then be available at [http://localho
 ## Production build locally
 
 To run this application locally with a production build, run:
-`npm run build:local && npm run start`. 
+`npm run build && npm run start`. 
 
-We use `npm run build:local` locally which bundles the application pointing at localhost for data and static assets.
+We use `npm run build` locally which bundles the application pointing at localhost for data and static assets.
 
 To avoid indexing by search engines during our early development, there is a `nofollow` page level meta tag in `Document.jsx`.
 
+## Using environment builds locally
+
+This is mainly used for debugging `latest` using the TEST and LIVE environment bundles. Ensure that the bundles exist in the static asset location for the correct environment before starting to debug.
+
+To run TEST bundles on localhost:
+`rm -rf build && npm run build:test && npm run start`
```

Comment:
You need to have `NODE_ENV=production` for the build step for this to work as expected and switch to use the `SIMORGH_PUBLIC_STATIC_ASSETS_PATH` for the Test/Live environment instead of localhost.

Reply:
Great spot, I've added it to the commands in the package.json